.github/workflows/test-suite-desktop-e2e.yml

```diff
@@ -0,0 +1,86 @@
+name: "[Build/Test] suite-desktop e2e"
+# this action builds a desktop version of Suite and runs basic test suite for it
+
+# run only if there are changes in suite or related libs paths
+on:
+  pull_request:
+    branches:
+      - develop
+    paths-ignore:
+      - "suite-native/**"
+      - "packages/connect*/**"
+      - "packages/react-native-usb/**"
+      # ignore unrelated github workflows config files
+      - ".github/workflows/connect*"
+      - ".github/workflows/suite-native*"
+      - ".github/workflows/build-desktop*"
+      - ".github/workflows/release*"
+      - ".github/workflows/template*"
+
+jobs:
+  run-desktop-tests:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - TEST_FILE: "spawn-bridge"
+            CONTAINERS: "trezor-user-env-unix"
+          - TEST_FILE: "suite-guide"
+            CONTAINERS: "trezor-user-env-unix"
+          - TEST_FILE: "wallet-discovery"
+            CONTAINERS: "trezor-user-env-unix"
+          # - TEST_FILE: "cardano-discovery"
```

Comment:
unrelated but what is the difference between wallet-discovery and cardano-discovery?